tests/test_accelerator.py

```diff
@@ -156,9 +156,8 @@ def test_save_model_offload(self, use_safetensors):
         model = ModelForTest()
```

Comment:
Let's compare the output to have a better test 

```suggestion
        inputs = torch.randn(3, 3)
        model = ModelForTest()
        expected = model(inputs)
```